infrastructure/src/main/java/org/corfudb/infrastructure/IServerRouter.java

```diff
@@ -210,7 +211,8 @@ default void sendWrongEpochError(HeaderMsg requestHeader, ChannelHandlerContext
         sendResponse(response, ctx);
 
         log.trace("sendWrongEpochError[{}]: Incoming request received with wrong epoch, got {}, expected {}, " +
-                "request was {}", requestHeader.getRequestId(), requestHeader.getEpoch(), correctEpoch, requestHeader);
+                "request was {}", requestHeader.getRequestId(), requestHeader.getEpoch(),
```

Comment:
can you wrap this with a `if (log.isTraceEnabled())`

Reply:
Added, thanks!